mpqp/core/instruction/measurement/basis_measure.py

```diff
@@ -83,6 +83,17 @@ def to_other_language(
                 return Measure()
             else:
                 raise NotImplementedError(f"{type(self.basis)} not handled")
+        if language == Language.QASM2:
```

Comment:
I think we should check that the computational basis is used

Reply:
done

Reply:
I would do something like 
```py
return measure.pre_measure().to_other_language(language) + f"\nmeasure q[{target}] c[{i}];"
```

Alternatively, if we already cover the basis change where this function is called, we should raise a warning in case the user calls this function themself. Something like `"measure in non computational basis, don't forget about adding the basis change circuit before measuring"`. And potentially a way to filter out this warning when we call the function ourselves.